src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -109,7 +112,6 @@ public ValidatingOutputFormat getOutputFormatForRun(BatchSinkContext context) th
 
   @Override
   public void prepareRun(BatchSinkContext context) throws Exception {
-    super.prepareRun(context);
```

Comment:
Better to call super.prepareRun() first

Reply:
If I call it first, i'm afraid it will use the default `LineageRecorder` instead of the overridden one.

Reply:
Why? The parent class should also be calling the `getLineageRecorder` method. Because this class overrides this method, this will call the `getLineageRecorder` method in this class.